.github/ISSUE_TEMPLATE/bug_report_form.yml

```diff
@@ -7,34 +7,26 @@ body:
     attributes:
       value: |
         Thanks for taking the time to report a bug in the Oppia project.
-  - type: checkboxes
+  - type: markdown
     attributes:
-      label: Is there an existing issue for this?
-      description: Before filing a new issue, please do a quick search to check that it hasn't already been filed on the [issue tracker](https://github.com/oppia/oppia/issues).
-      options:
-      - label: I have searched the existing issues
-        required: true
+      value: |
+        Before filing a new issue, please do a quick search to check that it hasn't already been filed on the [issue tracker](https://github.com/oppia/oppia/issues) by typing the issue in the search box next to the "New issue" button.
```

Comment:
Before filing a new issue, **please do a quick search** to check that it hasn't already been filed on the issue tracker. You can do this by going to [this link](...) and typing some words related to the issue in the search box next to ...

Reply:
Done